notebooks/vertex_genai/solutions/grounding_vertex_agent_builder.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>I am not seeing any <code>tools</code> keyword arg in the cell below, only a <code>goundig_source</code> arg. Typo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>